core/src/test/java/org/apache/iceberg/TestFastAppend.java

```diff
@@ -214,7 +215,8 @@ public void testFailure() {
     Assert.assertFalse("Should clean up new manifest", new File(newManifest.path()).exists());
   }
 
-  @Test
```

Comment:
This test is no longer valid as I think we should not delete manifests passed to us. It should be the responsibility of the caller.